tools/deb/envoy_bootstrap_v2.json

```diff
@@ -56,7 +56,17 @@
         "patterns": [
           {{- range $a, $s := .inclusionPatterns }}
             {
-              "prefix": "{{$s}}"
+              "prefix": "{{js $s}}"
```

Comment:
@mandarjog I have a question about this line and need your wisdom.  I added the "jq" because I discovered that without it if the annotation value included backslashes it wouldn't generate.  This means that the behavior will slightly change between Istio 1.1 and 1.2.  I don't expect any users to encounter the change, but what do you think?